roscala/src/main/scala/coop/rchain/rosette/prim/ctxt.scala

```diff
@@ -0,0 +1,43 @@
+package coop.rchain.rosette.prim
+
+import cats.implicits._
+import coop.rchain.rosette._
+import coop.rchain.rosette.macros.{checkArgumentMismatch, checkTypeMismatch}
+import coop.rchain.rosette.{Ctxt, CtxtTransition, Ob}
+import coop.rchain.rosette.Ob.Lenses._
+
+object ctxt {
+  object ctxtRtn extends Prim {
+    override val name: String = "ctxt-rtn"
+    override val minArgs: Int = 2
+    override val maxArgs: Int = 2
+
+    override def fn: CtxtTransition[PrimResult] =
+      for {
+        ctxt      <- getCtxt
+        globalEnv <- getGlobalEnv
+        optArg0   = ctxt.arg(0).map(_.asInstanceOf[Ctxt])
+        optArg1   = ctxt.arg(1)
+
+        result <- (optArg0, optArg1) match {
+                   // Both ARG(0) and ARG(1) are available
+                   case (Some(arg0), Some(arg1)) =>
```

Comment:
You can use `(arg0, arg1) mapN { .. }` instead.

Reply:
Done.